Doc/library/socket.rst

```diff
@@ -1678,24 +1679,25 @@ the interface::
 
    import socket
 
-   # the public network interface
-   HOST = socket.gethostbyname(socket.gethostname())
 
-   # create a raw socket and bind it to the public interface
-   s = socket.socket(socket.AF_INET, socket.SOCK_RAW, socket.IPPROTO_IP)
-   s.bind((HOST, 0))
+    # the public network interface
```

Comment:
Same there. `# the public network interface` should start at the same level of indentation as `import socket` (line 1680)